src/app/shared/theme-chooser/theme-chooser.html

```diff
@@ -0,0 +1,16 @@
+<button md-icon-button [md-menu-trigger-for]="themeMenu">
+  <md-icon>format_color_fill</md-icon>
+</button>
+
+<md-menu class="theme-chooser-menu" #themeMenu="mdMenu" x-position="before">
```

Comment:
I pulled this down to try it out and found that the keyboard shortcuts for the menu don't work.